CONTRIBUTING.md

```diff
@@ -69,6 +69,9 @@ opensource-codeofconduct@amazon.com with any additional questions or comments.
 ## Security issue notifications
 If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public github issue.
 
+## Troubleshooting
+
+* The local **API reference** throws a `Module aws_lambda_powertools not found` error when an exception is raised in the code-base, when you run `poetry run pdoc --pdf aws_lambda_powertools` the exception is shown.
```

Comment:
is there a fix for it? Because it's under Troubleshooting, I thought there was :)

Reply:
Ah I will rephrase it

What I ment to write is that if the local api webserver doesn't work and throwing 👆 that error. You can then use the pdf generation option to display the exception that is raised.

Reply:
Ahhh, gotcha! I only saw that error when the package wasn't installed locally (`make dev`).

Reply:
Ah nice I added the `make dev` as a solution as well, the `pdoc3` path can't be wrong anymore since we run everything in a poetry shell right?